example/vectorAdd/src/vectorAdd.cpp

```diff
@@ -85,7 +85,8 @@ auto main() -> int
     using QueueAcc = alpaka::Queue<Acc, QueueProperty>;
 
     // Select a device
-    auto const devAcc = alpaka::getDevByIdx<Acc>(0u);
+    auto const platform = alpaka::Pltf<Acc>{};
+    auto const devAcc = alpaka::getDevByIdx(platform, 0);
```

Comment:
We can no longer get the nth device of an accelerator, because the creation of a platform object is now necessary.